xsnap/makefiles/win/xsnap.mak

```diff
@@ -30,6 +30,9 @@ C_OPTIONS = \
 	/I$(PLT_DIR) \
 	/I$(SRC_DIR) \
 	/I$(TLS_DIR) \
+	/I$(MODDABLE)/modules/data/text/decoder \
+	/I$(MODDABLE)/modules/data/text/encoder \
+	/I$(MODDABLE)/modules/data/base64 \
```

Comment:
Did you happen to test the windows build? I avoid writing untested code, and I Don't Do Windows, so I have not maintained `makefiles/win`.